yt_dlp/extractor/tvp.py

```diff
@@ -297,12 +315,14 @@ def _real_extract(self, url):
 class TVPEmbedIE(InfoExtractor):
     IE_NAME = 'tvp:embed'
     IE_DESC = 'Telewizja Polska'
+    # XFF is not effective
+    _GEO_BYPASS = False
```

Comment:
```suggestion
    _GEO_BYPASS = False
```
That's the only reason this is ever set